src/code/qualification/initializeQualificationWorkflow.m

```diff
@@ -86,7 +86,7 @@
         % ObservedDataSets format to be determined
         % Temporary format: structure where the observed data is a table y
         if isfield(ConfigurationPlan.ObservedDataSets{i}, 'Type')
-            ObsTable = readtable(fullfile(REInput_path, ConfigurationPlan.ObservedDataSets{i}.Path), 'Encoding', 'UTF-8');
+            ObsTable = readtable(fullfile(REInput_path, ConfigurationPlan.ObservedDataSets{i}.Path), 'Encoding', 'system');

```

Comment:
this change solved the particular problem with observed data described in the referenced issue
(on my comp with german language settings and on another computer with english lang. settings).
This would theoretically make the hack above not necessary anymore.

however not sure if we will not get problems in other cases. If so, this change might be reverted